package.json

```diff
@@ -7,29 +7,44 @@
   "license": "MIT",
   "scripts": {
     "build": "webpack --mode=production",
-    "dev": "webpack --mode=development --watch --progress"
+    "dev": "webpack --mode=production --watch"
```

Comment:
why production in dev?

Reply:
We do not have a dev mode in this context, I mean is not configured